topics/videoGames.html

```diff
@@ -7,13 +7,17 @@
     <title>Video Game Recommendations</title>
 </head>
 <body>
-    <h1>Video Game Recommendations from Anthony Nardiello</h1>
+    <h1>Video Game Recommendations</h1>
```

Comment:
dropped the name as these are combined recommendations - github will tell us who edited what line via git blame